plugins/BEdita/Core/src/Model/Table/ProfilesTable.php

```diff
@@ -110,9 +110,9 @@ public function beforeSave(Event $event, EntityInterface $entity)
         if (empty($entity->get('email'))) {
             $entity->set('email', null);
         }
-        if (!$entity->has('title') && ($entity->has('name') || $entity->has('surname'))) {
+        if ($entity->get('title') === '' && (!empty($entity->get('name')) || !empty($entity->get('surname')))) {
```

Comment:
If `$entity->get('title') === 'null'`, the title is left unchanged. Is this intentional?

Reply:
you mean `=== null` ?

at first I used `empty($entity->get('title'))` condition, but, as suggested by @batopa, we probably need a way to be able to reset `title`. So if `title` is `null` it stays `null`

I know it's probably not so intuitive...